oracle-linux-image-tools/cloud/ovm/image-scripts.sh

```diff
@@ -53,20 +53,21 @@ cloud::image_cleanup() {
 #   None
 #######################################
 cloud::image_package() {
-  local mk_envelope="${CLOUD_DIR}/${CLOUD}/mk-envelope.sh"
+  commmon::convert_to_vmdk System.vmdk
```

Comment:
Too many `m` in `common`...

```suggestion
  common::convert_to_vmdk System.vmdk
```

Reply:
Thank you!